package.json

```diff
@@ -94,6 +94,7 @@
     "critters": "0.0.6",
     "cross-env": "6.0.3",
     "cross-spawn": "6.0.5",
+    "es5-ext": "0.10.53",
```

Comment:
Why was this added to the root `package.json`? It doesn't look like it's being used in tests or build files